mac

```diff
@@ -36,42 +40,178 @@ fi
 
 fancy_echo "Updating Homebrew formulae ..."
 brew update --force # https://github.com/Homebrew/brew/issues/1151
-brew bundle --file=- <<EOF
-tap "thoughtbot/formulae"
-tap "homebrew/services"
-tap "homebrew/homebrew-cask"
+
+fancy_echo "Adding taps..."
+brew tap "thoughtbot/formulae"
+brew tap "homebrew/services"
+brew tap "homebrew/homebrew-cask"
+brew tap "homebrew/cask-fonts"
+
+fancy_echo "Installing formulae..."
 
 # Unix
-brew "git"
-brew "openssl"
+brew install "zsh" 
+# newer than macOS bundled
+
+# https://the.exa.website/
+brew install "exa"
```

Comment:
Cool but perhaps a bit more personal, install happens behind the scenes so user might not even know this is available

Reply:
it's aliased to ls in the zsh-quickstart, so what they will see is a more configurable and colorful output with file icons:

<img width="933" alt="image" src="https://user-images.githubusercontent.com/595328/214140408-dd2f3b2b-e14d-4817-b799-8990283a1f5e.png">